homeassistant/components/apcupsd/sensor.py

```diff
@@ -34,392 +34,343 @@
     "alarmdel": SensorEntityDescription(
         key="alarmdel",
         translation_key="alarm_delay",
-        name="UPS Alarm Delay",
     ),
     "ambtemp": SensorEntityDescription(
         key="ambtemp",
-        name="UPS Ambient Temperature",
+        translation_key="ambient_temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "apc": SensorEntityDescription(
         key="apc",
         translation_key="apc_status",
-        name="UPS Status Data",
         entity_registry_enabled_default=False,
     ),
```

Comment:
What values can come from this sensor?

Reply:
We should probably also deprecate this sensor tbh. 

An example value is `001,036,0876` which shows "[version, number of records and number of bytes following](https://linux.die.net/man/8/apcaccess#:~:text=apcaccess%20is%20a%20program%20which,installed%2C%20properly%20configured%20and%20running.)" that isn't really useful for the end users anyway.